src/style.css

```diff
@@ -1,3 +1,78 @@
+* {
+  margin: 0;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  background-color: #fff;
+  color: #00acdc;
+}
+
+header {
+  min-width: 1024px;
+  max-width: 1200px;
+  flex-shrink: 0;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
```

Comment:
Do we have something similar in the mockup?
Please do everything according to the mockup